delta/sourcing-psql/src/main/resources/scripts/schema.ddl

```diff
@@ -61,4 +61,13 @@ CREATE TABLE IF NOT EXISTS public.scoped_states(
     PRIMARY KEY(type, org, project, id, tag),
     UNIQUE (org, project, id, tag)
 );
-CREATE INDEX IF NOT EXISTS scoped_states_ordering_idx ON public.scoped_states USING BRIN (ordering);
\ No newline at end of file
+CREATE INDEX IF NOT EXISTS scoped_states_ordering_idx ON public.scoped_states USING BRIN (ordering);
+
+--
+-- Table for projection offsets
+--
+CREATE TABLE if NOT EXISTS public.projection_offsets(
+    name   text,
```

Comment:
It can be added later but it would be nice to have:
* When it got created
* When it was last saved 
* For views, to have the project/id so that we can fetch them with this columns and avoid to have to fetch the view to build the name to fetch content